jboss-integration/src/main/java/org/teiid/jboss/rest/RestASMBasedWebArchiveBuilder.java

```diff
@@ -506,14 +602,29 @@ private void buildQueryProcedure(String vdbName, int vdbVersion, String modelNam
 			}
 			{
 			av0 = mv.visitAnnotation("Ljavax/ws/rs/Path;", true);
-			av0.visit("value", "/query");
+			av0.visit("value", modelName);
```

Comment:
Why change to modelname? This is a generic service, that one can execute, so need to have a generic name